src/backend/common/queries/dict_converters/event_details_converter.py

```diff
@@ -52,7 +41,7 @@ def eventDetailsConverter_v3(cls, event_details: EventDetails) -> EventDetailsDi
             "insights": event_details.insights
             if event_details
             else {"qual": {}, "playoff": {}},
-            "oprs": normalized_oprs if normalized_oprs else {},  # OPRs, DPRs, CCWMs
+            "oprs": event_details.matchstats,
```

Comment:
Will this always be non-null, or do we need to keep this check?

Reply:
Truthfully I don't know enough to say it will be always non-null - however, the previous check didn't make a ton of sense to me, as `normalized_oprs` was always non-null anyway: https://github.com/the-blue-alliance/the-blue-alliance/blob/py3/src/backend/common/queries/dict_converters/event_details_converter.py#L30

Reply:
Confirmed in our database that this CAN be null. My quick spot-check was `2000cmp`, which is null. So yes - let's make sure to add a null check here.